gensrc/thrift/MasterService.thrift

```diff
@@ -61,6 +61,7 @@ struct TTabletInfo {
     16: optional Types.TVersion min_readable_version
     17: optional i64 binlog_config_version
     18: optional bool is_error_state
+    19: optional i64 max_rowset_creation_time
```

Comment:
I think max_rowset_create_time (creation -> create) is more reasonable.

Reply:
I think we should keep this name? because it is consistent with the BE side.